Offseason2022/src/main/java/frc/robot/commands/ClimberTimerOverride.java

```diff
@@ -3,23 +3,31 @@
 
 package frc.robot.commands;
 
+import edu.wpi.first.wpilibj.XboxController;
 import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.RobotContainer;
+import frc.robot.subsystems.Climber;
 
 /**
  *
  */
 public class ClimberTimerOverride extends CommandBase
 {
-  public ClimberTimerOverride( )
+  private Climber        m_climber;
+  private XboxController operatorController;
+
+  public ClimberTimerOverride(Climber climber)

```

Comment:
We will be modifying all of our "simple" commands to pass (inject) all subsystems in as parameters. So in addition to the Climber subsystem, passing the second parameter as "XboxController gamepad" would be more direct than using RobotContainer.getInstance(). Try to add this second parameter and see if you can simplify the references to the operator controller.